draft-irtf-cfrg-hpke.md

```diff
@@ -505,21 +514,22 @@ in {{kem-ids}}.
 
 The `psk` and `pskID` fields MUST appear together or not at all.
 That is, if a non-default value is provided for one of them, then
-the other MUST be set to a non-default value.
+the other MUST be set to a non-default value. This requirement is
+encoded in the `VerifyPSKInputs` function below.
 
 The `psk`, `pskID`, and `info` fields have maximum lengths that depend
 on the KDF itself, on the definition of `LabeledExtract`, and on the
 constant labels used together with them. See {{kdf-input-length}} for
 precise limits on these lengths.
 
-The key and nonce computed by this algorithm have the property that
-they are only known to the holder of the recipient private key, and
-the party that ran the KEM to generate `zz` and `enc`.  If the `psk`
-and `pskID` arguments are provided, then the recipient is assured
-that the sender held the PSK.  If the `pkSm` argument is
-provided, then the recipient is assured that the sender held the
-corresponding private key (assuming that `zz` and `enc` were
-generated using the AuthEncap / AuthDecap methods; see below).
+The `key`, `nonce`, and `exporter_secret` computed by this algorithm have the
+property that they are only known to the holder of the recipient private
+key, and the entity that used the KEM to generate `zz` and `enc`.
+If the AuthEncap function of the KEM was used to generate `zz` and `enc`,
+the recipient is assured that the sender held the private key `sKS`.
```

Comment:
This isn't quite true (per the KCI problems noted in our offline thread) -- can we adjust this?